### DISCUSSION TITLE
Type safe routing

### DISCUSSION THREAD

**Discussion by MCole-K9** [sveltejs/kit#9200] [Ideas]
Nextjs 13.2 just shipped with typesafe routing and it reminded me that for a while I've wanted this in Sveltekit, also wanted named routes (Django Style ) but that's not necessary right now. 
For the Typesafe Routing,
Since native anchor tags are being used, could we take like a Tailwind Intellisense approach and target the href on anchor tags or some other means. Did a quick but not thorough glance at the discussions , and haven't seen this so forgive me if it was already mentioned

**Comment by geoffrich** [comment]
There's some rather lengthy previous discussion in https://github.com/sveltejs/kit/issues/647 if you hadn't already seen it.

**Reply by MCole-K9** [reply]
Okay. Thanks
